tests/protocol/test_connectors.py

```diff
@@ -190,6 +190,9 @@
 
 SYNC_CURSOR = {"foo": "bar"}
 
+INDEX_NAME = "index_name"
+ACCESS_CONTROL_INDEX_PREFIX = "search-acl-filter-"
```

Comment:
Shouldn't this be imported?

Reply:
I normally always patch constants, so I'm not depending on their concrete "real" value anywhere in the test.